mvp.feature

```diff
@@ -1151,7 +1163,7 @@ Feature: MVP features for NVA
   # Actions from Page : My Worklist
   @1252
   Scenario: Curator expands an item in the Worklist
-    Given that a Curator views DOI Request tab
+    Given that a Curator views DOI Request tab (Part II)
```

Comment:
Is there a tab called `DOI Request`? If it exists we should refer to it. Otherwise we can use a phrasing similar to the one on previous scenarios.
```
Given that a Curator views the DOI Request tab
```
or 
```
Given that a Curator has clicked the tab DOI request 
```

Reply:
Yes, there is a tab called `DOI Request`. 
The Given statement should refer to a scenario, and NP-1375 is called `Curator views DOI Request tab (Part II)`

Reply:
Each scenario should be independent. 
I do not understand why the Given statement should refer to a scenario. A `Given` statement is a condition, an action that has already been completed, such as the click of a button or the click on tab. A scenario cannot be a Given because it is test, it is not one status. In any case Given <scenario title> will not run the mentioned scenario.

I am not sure what is the purpose of the Scenario reference in that scenario. Is it possible that a Background could help?